model/resource/process.yaml

```diff
@@ -22,7 +23,9 @@ groups:
       - ref: process.command_args
         requirement_level:
           conditionally_required: See [Selecting process attributes](#selecting-process-attributes) for details.
-      - ref: process.owner
+      - ref: process.status
+        requirement_level:
+          conditionally_required: See [Selecting process attributes](#selecting-process-attributes) for details.
```

Comment:
I think `Selecting process attributes` applies to things that identify the process, I guess status is not one of them and would be `recommended: when applicable` or just `recommended`?